templates/book-section.html

```diff
@@ -3,6 +3,10 @@
 
 {% block head_extensions %}
 <script src="/optional-helpers.js"></script>
+<!-- When this javascript is loaded and initialized it will look for divs with the class `mermaid`
+it will replace the content of those divs with the generated mermaid content-->
+<script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
```

Comment:
We shouldn't block page paint on mermaid loading and being parsed.

Reply:
I've removed the block-on-load